ops/ansible/playbooks-ccs/roles/platinum/tasks/install-dependencies.yml

```diff
@@ -3,8 +3,8 @@
 # Builds a 'Platinum' AMI (updates) and installs program specific dependencies
 # from a base Gold Image in the CCS AWS Environment
 ##
-- name: Enable java11 repository
-  command: amazon-linux-extras enable java-openjdk11
+- name: Enable java17 repository
+  command: amazon-linux-extras enable java-17-amazon-corretto-devel
```

Comment:
Pretty sure we dont need the enable command anymore on java17, but not sure how to safely remove this.

Reply:
Thanks!